src/app/task/task.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class TaskService {
+
+  allTasks: object[] = [];
+
+  updateTaskData(name: string) {      
```

Comment:
this method should define what type it returns;
rename this method with correct name